### PR TITLE
Issue 212: Extract help command logic to handler

### LINKED ISSUE #212: Extract help command logic to proper handler
## Task Context
As a next steps when working on project refactoring, we decided to extract the code that is handling specific commands to separate classes. 
This task is about extracting current `help` command logic from `HydrateReminderPlugin` class to `HelpCommandHandler` class.

## Acceptance Criteria / Goals
- `help` command logic is extracted to dedicated class `HelpCommandHandler`
- `HydrateReminderPlugin::onCommandExecuted` should call `commandDelegate.invokeCommand(commandExecuted)` when the selected command is `help`
- Solution was tested using unit tests

## Additional Notes
Please refer to [CONTRIBUTING.md](https://github.com/jmakhack/hydrate-reminder/blob/master/CONTRIBUTING.md) for info on how to setup, build, run, and test this project. Feel free to ping @jmakhack or @mslowiak in the comments below with any questions if needed. Also, join the [Discord](https://discord.gg/RTgxfFW9mS) to meet and engage with other contributors!

[](https://discord.gg/RTgxfFW9mS)

### PR DESCRIPTION
**Pull request by mslowiak**
## Associated Issue
Closes #212 

## Implemented Solution
Command logic was extracted to proper handler class

### CODE REVIEW

File: src/main/java/com/hydratereminder/HydrateReminderPlugin.java
Comment (jmakhack) [inline src/main/java/com/hydratereminder/HydrateReminderPlugin.java:294]: just a small nitpick: can you split this out into two lines for easier readability?

Everything else looks good. I'll merge this in after the change and merge conflict resolutions.

Comment (mslowiak) [reply src/main/java/com/hydratereminder/HydrateReminderPlugin.java]: @jmakhack resolved, should be ok now :)

**Review by jmakhack — APPROVED** [review]

**Pull request merged**
